bigtable/client/table.h

```diff
@@ -226,6 +249,9 @@ class Table {
    *     Both rules accept the family and column identifier to modify.
    * @param rules is the zero or more ReadModifyWriteRules to apply on a row.
    * @returns modified row
+   *
+   * **Examples**
```

Comment:
Is this supposed to be singular or plural? Other examples say "Example", this one says "Examples" even though it points to a single snippet.

Reply:
Fixed.